src/User.js

```diff
@@ -1,5 +1,7 @@
+import Hydration from './Hydration';
+
```

Comment:
related to comments for hydration class file

Reply:
In scripts on lines 27 - 55 The API data is being fetched, and is only available inside of the .then(), or if we used an event listener somehow to access that data when it is "resolved" 

What I was finding was that because the data has to stay inside of the initializer function, I couldn't make it available to the classes globally, and I don't think we can import it. I also tried creating a variable inside of the initializer function that I then passed into the User class, but was still running into trouble. 

I would try cloning the API repo into it's own directory (if you haven't) and running that server with npm start and then running the local host server from another terminal window and see if you can get the hydration data to flow without the hard data files. 

If so that would be awesome, because your refactor was awesome and you did a really good job with that!!